src/main/resources/static/built/css/fixes.css

```diff
@@ -1,3 +1,3 @@
 .rc-slider-tooltip {
-    z-index: 1000;
+    z-index: 2000;
 }
```

Comment:
can you tack an extra line on the bottom of this file?

Reply:
I can add an extra line on the bottom. What do you want to add?

Reply:
was just a style preference to have 1 new line return at the bottom of each file.  Sorry nitpicky stuff

Reply:
Sorry, I am not understanding what you want. 

You want to add a line to each file to return the css?

Reply:
no just a new line at the end of the file.  It stops github from putting an annoying red symbol